<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>file_path_find_frame</class>
 <widget class="QFrame" name="file_path_find_frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>248</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QPushButton" name="file_path_find_btn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>찾아보기</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="file_path_find_tBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exe_btn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>195</red>
         <green>240</green>
         <blue>237</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>77</red>
         <green>227</green>
         <blue>165</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>195</red>
         <green>240</green>
         <blue>237</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>77</red>
         <green>227</green>
         <blue>165</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>195</red>
         <green>240</green>
         <blue>237</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>77</red>
         <green>227</green>
         <blue>165</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>변환</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Bottom-Up File</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sht_name_list">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sheet Name</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_allocation">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>96</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Allocation</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_datacube">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>96</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Data 생성</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>103</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="save_path_find_tBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>98</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="save_path_find_btn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>97</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>찾아보기</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="proc_bar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>451</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>106</number>
   </property>
   <property name="maximum">
    <number>146</number>
   </property>
   <property name="value">
    <number>106</number>
   </property>
  </widget>
  <widget class="QLabel" name="job_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="volume_ver">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume_Version</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Senario_Version</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="scenario_ver">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>148</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="vol_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
